api/v1alpha1/aws_clusterconfig_types.go

```diff
@@ -9,28 +9,32 @@ import (
 
 type AWSSpec struct {
 	// AWS region to create cluster in.
-	// +optional
+	// +kubebuilder:validation:Optional
 	Region *Region `json:"region,omitempty"`
+
 	// AWS network configuration.
-	// +optional
+	// +kubebuilder:validation:Optional
 	Network *AWSNetwork `json:"network,omitempty"`
-	// +optional
+
+	// +kubebuilder:validation:Optional
 	ControlPlaneLoadBalancer *AWSLoadBalancerSpec `json:"controlPlaneLoadBalancer,omitempty"`
 }
 
 type Region string
 
 type AWSNetwork struct {
-	// +optional
+	// +kubebuilder:validation:Optional
 	VPC *VPC `json:"vpc,omitempty"`
 
 	// AWS Subnet configuration.
-	// +optional
+	// +kubebuilder:validation:Optional
```

Comment:
Same as above, should be a pointer?

Reply:
This is a slice (aliased type) so is effectively already a pointer.